spec/scrabble_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'colorize'
+require 'spec_helper'
+require './lib/scrabble'
+
+#format documentation
+
+describe "Converting letters to values" do
+  context "Letters have values" do |variable|
```

Comment:
Quick note on naming conventions in Rspec: Usually the context/describe blocks are to describe a specific method. So in this case, since these tests are testing the `.score` class method, generally you might see `describe ".score" do`, and then the `it` blocks give a description of the expected functionality (which you do just fine here).